tsconfig.json

```diff
@@ -15,9 +15,6 @@
     "strict": false,
     "target": "ES2020"
   }, 
-  "exclude": [
```

Comment:
that may force .spec.ts files to be in the npm build package which we don't want...

Reply:
There are no .spec files in ./src, so this is safe to delete.